src/poliastro/earth/sensors.py

```diff
@@ -52,9 +54,9 @@ def min_and_max_ground_range(h, η_fov, η_center, R):
 def max_and_min_ground_range_with_specific_azimuth(
     h, η_fov, η_center, β, φ_nadir, λ_nadir, R
 ):
-    """
-    Calculates the difference in ground-range angles from the η_center angle and the latitude and longitude of the target
+    """Calculates the difference in ground-range angles from the η_center angle and the latitude and longitude of the target
     for a desired phase angle,:math:`{\beta}`, used to specify where the sensor is looking.
```

Comment:
This wont work :)

Reply:
@Yash-10 can you please revert this back?

Reply:
```suggestion
    for a desired phase angle, β, used to specify where the sensor is looking.
```

Reply:
Thanks..